### PR TITLE
TE-4334 raised minimum version of php to 7.2

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,9 +17,6 @@ matrix:
 
   fast_finish: true
 
-  allow_failures:
-    - php: 7.1
```
Comment (dereuromark) [inline .travis.yml]: I think this never had proper tests, thus the allow_failures for the current  (7.2) then.

Comment (RodinVasily) [reply .travis.yml]: but it doesn't fail on 7.2, so why would we allow_failures there?

Comment (dereuromark) [reply .travis.yml]: Yeah the problem is the single entry in PHP list :) Always needs min+max to be valid and running for those.

File: .travis.yml [hunk 2]
```diff
@@ -18,7 +19,7 @@ matrix:
   fast_finish: true
 
   allow_failures:
-    - php: 7.1
+    - php: 7.2
```
Comment (dereuromark) [inline .travis.yml]: Do we also need to exlude the 7.4 run?

**Pull request merged**